index.bs

```diff
@@ -549,15 +549,13 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 : <dfn>Authenticator</dfn>
 : <dfn>[WAA]</dfn>
-:: A cryptographic entity used by a [=[WAC]=] to (i) generate a [=public key credential=] and register it with a [=[RP]=],
-    and (ii) [=authentication|authenticate=] by potentially [=user verification|verifying the user=], and then 
-    cryptographically signing and returning, in the form of an [=Authentication Assertion=], 
-    a challenge and other data presented by a [=[WRP]=] (in concert with the [=[WAC]=]).
+:: A cryptographic entity, existing in hardware or software, that can [=registration|register=] a user with a given [=[RP]=] 
+    and later [=Authentication Assertion|assert possession=] of the registered [=public key credential=] when requested by the [=[RP]=]. 
```

Comment:
I don't think we want to loose the notion of user verification:
suggest:
...assert possession=] of the registered [=public key credential=], and optionally [=user verification|verifying the user=],  when requested by the [=[RP]=].